tutorcodejail/templates/codejail/apps/config/tutor.py

```diff
@@ -1,10 +1,16 @@
 """Module with the configuration of config classes."""
-from base import BaseConfig
+from codejailservice.config import DevelopmentConfig, ProductionConfig
 
 
-class DevConfig(BaseConfig):
-    """Class to use for development context that inherits from BaseConfig."""
+class DevelopmentConfig(DevelopmentConfig):
+    """Class to use for development context that inherits from DevelopmentConfig."""
 
```

Comment:
I would say one of the more important things for the config is to be able to override the [`CODE_JAIL`](https://github.com/eduNEXT/codejailservice/blob/f19aa2cba22ba24d4899c008a31f18680fe0baef/codejailservice/config.py#L12) dict in the base config.

But if I were to add a patch to change a value like:

```python
CODE_JAIL["limits"]["REALTIME"] = 3
```

I would get an error because `CODE_JAIL` isn't defined. A quick hack would be to set the variable like this:

```python
class DevelopmentConfig(DevelopmentConfig):
    """Class to use for development context that inherits from DevelopmentConfig."""
    CODE_JAIL = DevelopmentConfig.CODE_JAIL
    ...
```

Then it would be easier to add patches that only override certain settings.

Reply:
Thanks. I already add the fix 🙌